DuckDuckGo/AppDelegate.swift

```diff
@@ -45,6 +45,7 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
     
     func applicationDidBecomeActive(_ application: UIApplication) {
         startMigration()
+        TrackerLoader.shared.updateTrackers()
```

Comment:
Also note that this means it's difficult to test on a device where swiping up to turn off Airplane mode and then swiping away means the trackers are automatically downloaded.  Easier to test on the simulator though by putting your entire Mac in Airplane mode.